packages/common/src/lib/list/list.component.ts

```diff
@@ -136,11 +138,21 @@ export class ListComponent implements AfterViewInit, OnInit, OnDestroy {
     if (item !== undefined) {
       this.focus(item);
     }
+
```

Comment:
There is something similar in the entity-table. It could have been useful. See https://github.com/infra-geo-ouverte/igo2-lib/blob/master/packages/common/src/lib/entity/entity-table/entity-table-row.directive.ts#L113